controllers/suite_test.go

```diff
@@ -315,6 +316,10 @@ var _ = BeforeSuite(func() {
 	err = util.IndexFieldsForVSHandler(context.TODO(), k8sManager.GetFieldIndexer())
 	Expect(err).ToNot(HaveOccurred())
 
+	rateLimiter := workqueue.NewMaxOfRateLimiter(
+		workqueue.NewItemExponentialFailureRateLimiter(100*time.Millisecond, 100*time.Millisecond),
```

Comment:
Do we have a dependency between internal and the timeout use here? if we do maybe we should use something like `2 * interval`?

Reply:
No, tested with sample SDK separately there is no dependency, it can be the same value (as per my tests).